misc/cron/archive.php

```diff
@@ -24,10 +17,10 @@
     define('PIWIK_USER_PATH', PIWIK_INCLUDE_PATH);
 }
 
-define('PIWIK_ENABLE_DISPATCH', false);
 define('PIWIK_ENABLE_ERROR_HANDLER', false);
 define('PIWIK_ENABLE_SESSION_START', false);
-require_once PIWIK_INCLUDE_PATH . "/index.php";
+
+require_once PIWIK_INCLUDE_PATH . "/core/bootstrap.php";
```

Comment:
I'm not sure about this change because `index.php` contains logic to include a `bootstrap.php` (*not* `core/bootstrap.php`) that can be provided by users to define `PIWIK_DOCUMENT_ROOT` and `PIWIK_INCLUDE_PATH`… Not including `index.php` may mean that these overrides are ignored, it might break their setup.